demos/list.html

```diff
@@ -1200,6 +1335,27 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        function clickHandler(evt, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          if(evt.target === li) {
```

Comment:
Missing space between `if` and `(`

Also technically you only need to query for the checkbox inside the `if`